girder/utility/assetstore_utilities.py

```diff
@@ -24,6 +24,11 @@
 from girder import events
 
 
+assetstoreTable = {AssetstoreType.FILESYSTEM: FilesystemAssetstoreAdapter,
```

Comment:
Just FYI, I'll be out until Monday but I'll go through and make all the
requested changes then. Have a good weekend everyone!

On Thu, Jun 9, 2016 at 4:05 PM Roni Choudhury <roni.choudhury@kitware.com>
wrote:

> I prefer that style too but I've seen the other one in Python projects
> before. I'll change it unless @zachmullen objects.
>
>
> On Thu, Jun 9, 2016 at 2:58 PM Michael Fogleman <notifications@github.com>
> wrote:
>
>> In girder/utility/assetstore_utilities.py
>> <https://github.com/girder/girder/pull/1372#discussion_r66501093>:
>>
>> > @@ -24,6 +24,11 @@
>> >  from girder import events
>> >
>> >
>> > +assetstoreTable = {AssetstoreType.FILESYSTEM: FilesystemAssetstoreAdapter,
>>
>> Is this the preferred formatting for dicts in girder? I'd do this:
>>
>> assetstoreTable = {
>>     AssetstoreType.FILESYSTEM: FilesystemAssetstoreAdapter,
>>     AssetstoreType.GRIDFS: GridFsAssetstoreAdapter,
>>     AssetstoreType.S3: S3AssetstoreAdapter,
>> }
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/girder/girder/pull/1372/files/91b74f1930d5a4c381b47090446bc4fccef8f68b#r66501093>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ACxNJG_BzypnNN_IpfVNpbGUPBDMKNcMks5qKGIygaJpZM4IyQ3c>
>> .
>>
>